src/common/ar-io.ts

```diff
@@ -563,7 +565,7 @@ export class ArIOWritable extends ArIOReadable implements ArIOWriteContract {
 
   /**
    * @param target @type {string} The address of the account you want to transfer IO tokens to.
-   * @param qty @type {number} The amount of IO or mIO to transfer.
+   * @param qty @type {number | mIOToken} The amount of IO or mIO to transfer.
```

Comment:
is it possible for us to put a deprecated warning that number will be unavailable in the future? or is that overkill for this?